<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Introduction to empdata package • empdata</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Introduction to empdata package">
<meta property="og:description" content="empdata">
<meta name="robots" content="noindex">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">empdata</a>
        <span class="version label label-danger" data-toggle="tooltip" data-placement="bottom" title="In-development version">0.0.1.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/empdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/nikopech/empdata">
    <span class="fab fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="empdata_files/header-attrs-2.7/header-attrs.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Introduction to empdata package</h1>
            
      
      
      <div class="hidden name"><code>empdata.Rmd</code></div>

    </div>

    
    
<!-- # ```{r, include = FALSE} -->
<!-- # knitr::opts_chunk$set( -->
<!-- #   collapse = TRUE, -->
<!-- #   comment = "#>" -->
<!-- # ) -->
<!-- # ``` -->
<div id="package-overview" class="section level2">
<h2 class="hasAnchor">
<a href="#package-overview" class="anchor"></a>Package overview</h2>
<p><strong><code>empdata</code></strong> is a collection of metagenomic data originated from the <a href="https://www.earthmicrobiome.org/">Earth Microbiome Project</a> (EMP). The package contains a subset of 975 samples and 91,364 sequences of 150bp length, along with their taxonomy classification.</p>
<p>Full documentation regarding the EMP dataset can be found in the related article <em><a href="https://doi.org/10.1038/nature24621">A communal catalogue reveals Earth’s multiscale microbial diversity</a></em>.</p>
</div>
<div id="sample-collection" class="section level2">
<h2 class="hasAnchor">
<a href="#sample-collection" class="anchor"></a>Sample collection</h2>
<p>Samples were submitted by the global community of microbial ecologists for microbiome analysis. Each sample was collected fresh and, where possible, immediately frozen in liquid nitrogen and stored at –80 °C. Bulk samples (for example, soil, sediment, faeces) and fractionated bulk samples (for example, sponge coral surface tissue, centrifuged turbid water) were taken using microcentrifuge tubes. Swabs (BD SWUBE dual cotton swabs or similar) were used for biofilm or surface samples. Filters (Sterivex cartridges, 0.2 μm, Millipore) were used for water samples. Samples were sent to laboratories in the United States for DNA extraction and sequencing: water samples to Argonne National Laboratory, soil samples to Lawrence Berkeley National Laboratory (pre-2014) or Pacific Northwest National Laboratory (2014 onward), and faecal and other samples to the University of Colorado Boulder (pre-2015) or the University of California San Diego (2015 onward).</p>
</div>
<div id="metadata-curation" class="section level2">
<h2 class="hasAnchor">
<a href="#metadata-curation" class="anchor"></a>Metadata curation</h2>
<p>Sample metadata were collected in compliance with MIMARKS, <a href="https://www.ebi.ac.uk/ena">EBI</a>, and <a href="http://qiita.microbio.me">Qiita</a> standards, as described in the <a href="http://www.earthmicrobiome.org/protocols-and-standards/metadata-guide">EMP Metadata Guide</a> and they contain information for 6 categories:</p>
<ul>
<li>Publication study: <em>Title, Principal Investigator, DOI, Description, Study ID, EBI accession</em>
</li>
<li>Sequencing: <em>BarcodeSequence, LinkerPrimerSequence, PCR primers, Target gene, Target subfragment, Illumina technology, Extraction center, Run center, Run date</em>
</li>
<li>Host subject information: <em>host taxid, host common name, host scientific name, host taxonomy</em>
</li>
<li>Geographical information: <em>collection timestamp, country, latitude_deg, longitude_deg, depth_m, altitude_m, elevation_m</em>
</li>
<li>Physicochemical properties: <em>temperature, pH value, salinity, oxygen concentration, phosphate concentration, ammonium concentration, nitrate concentration, sulfate concentration</em>
</li>
<li>Ontologies: <em>ENVO ontologies, EMPO ontologies</em>
</li>
</ul>
<p>A detailed documentation regarding the sample metadata can be found <a href="https://nikopech.github.io/empdata/reference/sample_metadata.html">here</a>.</p>
<div id="the-emp-ontology" class="section level3">
<h3 class="hasAnchor">
<a href="#the-emp-ontology" class="anchor"></a>The EMP Ontology</h3>
<p>The EMP Ontology (EMPO) for microbial environments was devised to facilitate the present analysis while preserving interoperability and it was coordinated by the ENVO team. It contains annotations from ENVO, UBERON (metazoan anatomy), PO (plant ontology), <a href="http://purl.obolibrary.org/obo/fao.owl">FAO</a>(fungal anatomy ontology), and OMP (ontology of microbial phenotypes) and it consists of three-level categories:</p>
<ul>
<li>
<em>empo_1</em>: free-living or host-associated lifestyles</li>
<li>
<em>empo_2</em>: saline or non-saline (if free-living) or animal or plant (if host-associated)</li>
<li>
<em>empo_3</em>: microbial environments</li>
</ul>
<p>Full descriptions of the EMP Ontology categories can be found at <a href="http://www.earthmicrobiome.org/protocols-and-standards/empo" class="uri">http://www.earthmicrobiome.org/protocols-and-standards/empo</a>. The W3C Web Ontology Language (OWL) document is available at <a href="http://purl.obolibrary.org/obo/envo/subsets/envoEmpo.owl" class="uri">http://purl.obolibrary.org/obo/envo/subsets/envoEmpo.owl</a>.</p>
</div>
</div>
<div id="s-rrna-amplicon-sequencing" class="section level2">
<h2 class="hasAnchor">
<a href="#s-rrna-amplicon-sequencing" class="anchor"></a>16S rRNA amplicon sequencing</h2>
<p>DNA extraction and 16S rRNA amplicon sequencing were done based on the <a href="http://www.earthmicrobiome.org/protocols-and-standards/16s">EMP standard protocols</a>.</p>
</div>
<div id="tag-sequence-and-subsets" class="section level2">
<h2 class="hasAnchor">
<a href="#tag-sequence-and-subsets" class="anchor"></a>Tag sequence and subsets</h2>
<p>The overall EMP dataset is provided throughout the <a href="ftp://ftp.microbio.me/emp/release1/">FTP site</a> (release1) and it contains a total of 97 studies and 27,742 samples.</p>
<p>Sequence data were error-filtered and trimmed to three different sequence lengths:</p>
<ul>
<li>The length of the shortest sequencing run (90 bp)</li>
<li>Sequence data trimmed to 100 bp (greater sequence resolution but fewer samples)</li>
<li>Sequence data trimmed to 150 bp (greater sequence resolution but fewer samples)</li>
</ul>
<p>Each subset comprises the exact sequences instead of clustered operational taxonomic units (OTU).</p>
<p>Additionally, samples were randomly picked to give equal (as possible) representation across environments (EMPO level 3) and studies within those environments. The subsets contain 10,000, 5,000, and 2,000 samples (nested subsets).</p>
</div>
<div id="package-subset" class="section level2">
<h2 class="hasAnchor">
<a href="#package-subset" class="anchor"></a>Package subset</h2>
<p>For the <code>empdata</code> package development, the subset of 2,000 samples with 150-bp length was chosen. The related BIOM file was downloaded from the <a href="ftp://ftp.microbio.me/emp/release1/otu_tables/deblur/">FTP site</a> (<em>emp_deblur_150bp.subset_2k.rare_5000.biom</em>). Sample metadata were also downloaded from the <a href="ftp://ftp.microbio.me/emp/release1/mapping_files/">FTP server</a> (<em>emp_qiime_mapping_subset_2k.tsv</em>).</p>
<p>The metadata file was additionally filtered to contain only the samples from the downloaded BIOM file. The resulting dataset contains 975 samples with 91,364 sequences of 150bp length.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://biodataanalysisgroup.github.io/team/">Nikolaos Pechlivanis</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
