<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Sample metadata table — sample_metadata_sub2k • empdata</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Sample metadata table — sample_metadata_sub2k" />
<meta property="og:description" content="This is a meta-data/mapping file containing related information about samples originated from the Earth Microbiome Project." />


<meta name="robots" content="noindex">

<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">empdata</a>
        <span class="version label label-danger" data-toggle="tooltip" data-placement="bottom" title="In-development version">0.0.1.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/empdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/nikopech/empdata">
    <span class="fab fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Sample metadata table</h1>
    
    <div class="hidden name"><code>sample_metadata_sub2k.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This is a meta-data/mapping file containing related information about samples originated from the Earth Microbiome Project.</p>
    </div>

    <pre class="usage"><span class='va'>sample_metadata_sub2k</span></pre>


    <h2 class="hasAnchor" id="format"><a class="anchor" href="#format"></a>Format</h2>

    <p>Data frame with columns</p><dl>
<dt>#SampleID</dt><dd><p>unique sample identifier (leading hashtag is required by Qiime)</p></dd>
<dt>BarcodeSequence</dt><dd><p>barcode sequence</p></dd>
<dt>LinkerPrimerSequence</dt><dd><p>linker primer sequence</p></dd>
<dt>Description</dt><dd><p>sample description</p></dd>
<dt>host_subject_id</dt><dd><p>unique host or subject (can have multiple samples per host_subject_id)</p></dd>
<dt>study_id</dt><dd><p>study identifier</p></dd>
<dt>title</dt><dd><p>study title</p></dd>
<dt>principal_investigator</dt><dd><p>PI of the study</p></dd>
<dt>doi</dt><dd><p>Digital Object Identifier (DOI) or PubMed ID of primary publication</p></dd>
<dt>ebi_accession</dt><dd><p>European Bioinformatics Institute (EBI) European Nucleotide Archive (ENA) accession number if submitted</p></dd>
<dt>target_gene</dt><dd><p>name of gene amplified (e.g., 16S rRNA)</p></dd>
<dt>target_subfragment</dt><dd><p>name of subfragment of gene amplified (e.g., V4)</p></dd>
<dt>pcr_primers</dt><dd><p>amplicon primer sequences used</p></dd>
<dt>illumina_technology</dt><dd><p>model of Illumina sequencer</p></dd>
<dt>extraction_center</dt><dd><p>where the sample was physically extracted</p></dd>
<dt>run_center</dt><dd><p>where the sample was physically sequenced (CCME=U Colorado Boulder, ANL=Argonne Natl Lab, UCSDMI=UC San Diego, CGS-GL=Wash U, UCD=U Colorado Denver)</p></dd>
<dt>run_date</dt><dd><p>date the sample was physically sequenced</p></dd>
<dt>read_length_bp</dt><dd><p>median read length in bp across study after quality filtering</p></dd>
<dt>sequences_split_libraries</dt><dd><p>number of sequences after demultiplexing with split_libraries_fastq.py</p></dd>
<dt>observations_closed_ref_greengenes</dt><dd><p>number of observations in closed-reference Greengenes table</p></dd>
<dt>observations_closed_ref_silva</dt><dd><p>number of observations in closed-reference Silva table</p></dd>
<dt>observations_open_ref_greengenes</dt><dd><p>number of observations in open-reference Greengenes table</p></dd>
<dt>observations_deblur_90bp</dt><dd><p>number of observations in 90-bp Deblur table</p></dd>
<dt>observations_deblur_100bp</dt><dd><p>number of observations in 100-bp Deblur table</p></dd>
<dt>observations_deblur_150bp</dt><dd><p>number of observations in 150-bp Deblur table</p></dd>
<dt>emp_release1</dt><dd><p>samples with &gt;=1 sequences (split libraries) per sample</p></dd>
<dt>qc_filtered</dt><dd><p>samples with &gt;=1000 observations in CR-GG &amp; CR-Silva &amp; OR-GG &amp; Deblur-90 but excluding controls (all subsets are in this set)</p></dd>
<dt>subset_10k</dt><dd><p>10000 samples with &gt;=10000 CR-GG &amp; CR-Silva &amp; OR-GG and &gt;=5000 Deblur-90, randomly selected and evenly distributed across <code>empo_3</code> categories and then across studies</p></dd>
<dt>subset_5k</dt><dd><p>5000 samples with &gt;=10000 CR-GG &amp; CR-Silva &amp; OR-GG and &gt;=5000 Deblur-90, randomly selected and evenly distributed across <code>empo_3</code> categories and then across studies</p></dd>
<dt>subset_2k</dt><dd><p>2000 samples with &gt;=10000 CR-GG &amp; CR-Silva &amp; OR-GG and &gt;=5000 Deblur-90, randomly selected and evenly distributed across <code>empo_3</code> categories and then across studies</p></dd>
<dt>sample_taxid</dt><dd><p>sample NCBI taxonomy ID</p></dd>
<dt>sample_scientific_name</dt><dd><p>sample NCBI scientific name looked up from taxonomy ID</p></dd>
<dt>host_taxid</dt><dd><p>host NCBI taxonomy ID</p></dd>
<dt>host_common_name_provided</dt><dd><p>host common name provided in the original mapping file</p></dd>
<dt>host_common_name</dt><dd><p>host NCBI common name looked up from taxonomy ID</p></dd>
<dt>host_scientific_name</dt><dd><p>host NCBI scientific name looked up from taxonomy ID</p></dd>
<dt>host_superkingdom</dt><dd><p>host superkingdom looked up from taxonomy ID</p></dd>
<dt>host_kingdom</dt><dd><p>host kingdom looked up from taxonomy ID</p></dd>
<dt>host_phylum</dt><dd><p>host phylum looked up from taxonomy ID</p></dd>
<dt>host_class</dt><dd><p>host class looked up from taxonomy ID</p></dd>
<dt>host_order</dt><dd><p>host order looked up from taxonomy ID</p></dd>
<dt>host_family</dt><dd><p>host family looked up from taxonomy ID</p></dd>
<dt>host_genus</dt><dd><p>host genus looked up from taxonomy ID</p></dd>
<dt>host_species</dt><dd><p>host species looked up from taxonomy ID</p></dd>
<dt>collection_timestamp</dt><dd><p>date and time when sample was collected</p></dd>
<dt>country</dt><dd><p>country where sample was collected</p></dd>
<dt>latitude_deg</dt><dd><p>latitude in degrees</p></dd>
<dt>longitude_deg</dt><dd><p>longitude in degrees</p></dd>
<dt>depth_m</dt><dd><p>depth in meters of sample below surface (earth surface if soil, sea/lake bottom if sediment, lake surface if lake, sea level if marine)</p></dd>
<dt>altitude_m</dt><dd><p>height above surface, usually zero unless the mouse is levitating (either depth or altitude can have a non-zero value but not both)</p></dd>
<dt>elevation_m</dt><dd><p>height above sea level in meters (from georeferencing tool)</p></dd>
<dt>env_biome</dt><dd><p>ENVO biome</p></dd>
<dt>env_feature</dt><dd><p>ENVO feature</p></dd>
<dt>env_material</dt><dd><p>ENVO material</p></dd>
<dt>envo_biome_0</dt><dd><p>level 0 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_1</dt><dd><p>level 1 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_2</dt><dd><p>level 2 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_3</dt><dd><p>level 3 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_4</dt><dd><p>level 4 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_5</dt><dd><p>level 5 ENVO biome looked up from given ENVO biome</p></dd>
<dt>empo_0</dt><dd><p>level 0 EMPO habitat from EMPO ontology</p></dd>
<dt>empo_1</dt><dd><p>level 1 EMPO habitat from EMPO ontology</p></dd>
<dt>empo_2</dt><dd><p>level 2 EMPO habitat from EMPO ontology</p></dd>
<dt>empo_3</dt><dd><p>level 3 EMPO habitat from EMPO ontology</p></dd>
<dt>adiv_observed_otus</dt><dd><p>observed tag sequences of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>adiv_chao1</dt><dd><p>Chao1 index of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>adiv_shannon</dt><dd><p>Shannon index of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>adiv_faith_pd</dt><dd><p>Faith's phylogenetic diversity of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>temperature_deg_c</dt><dd><p>ambient temperature in degrees Celcius</p></dd>
<dt>ph</dt><dd><p>pH value</p></dd>
<dt>salinity_psu</dt><dd><p>salinity in practical salinity units (PSU)</p></dd>
<dt>oxygen_mg_per_l</dt><dd><p>oxygen concentration in mg/L</p></dd>
<dt>phosphate_umol_per_l</dt><dd><p>phosphate concentration in µmol/L</p></dd>
<dt>ammonium_umol_per_l</dt><dd><p>ammonium concentration in µmol/L</p></dd>
<dt>nitrate_umol_per_l</dt><dd><p>nitrate concentration in µmol/L</p></dd>
<dt>sulfate_umol_per_l</dt><dd><p>sulfate concentration in µmol/L</p></dd>

</dl>

    <h2 class="hasAnchor" id="source"><a class="anchor" href="#source"></a>Source</h2>

    <p><a href='https://earthmicrobiome.org/'>https://earthmicrobiome.org/</a></p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>  <span class='fu'><a href='https://rdrr.io/r/utils/str.html'>str</a></span><span class='op'>(</span><span class='va'>sample_metadata_sub2k</span><span class='op'>)</span>
</div><div class='output co'>#&gt; Classes 'data.table' and 'data.frame':	2000 obs. of  76 variables:
#&gt;  $ #SampleID                         : chr  "550.L1S116.s.1.sequence" "550.L1S119.s.1.sequence" "550.L1S164.s.1.sequence" "550.L1S194.s.1.sequence" ...
#&gt;  $ BarcodeSequence                   : chr  "ATGCCTGAGCAG" "CAGCACTAAGCG" "ATGTACGGCGAC" "CGAAGACTGCTG" ...
#&gt;  $ LinkerPrimerSequence              : chr  "GTGCCAGCMGCCGCGGTAA" "GTGCCAGCMGCCGCGGTAA" "GTGCCAGCMGCCGCGGTAA" "GTGCCAGCMGCCGCGGTAA" ...
#&gt;  $ Description                       : chr  "sample_20 stool" "sample_23 stool" "sample_73 stool" "sample_105 stool" ...
#&gt;  $ host_subject_id                   : chr  "F4" "F4" "M3" "M3" ...
#&gt;  $ study_id                          : int  550 550 550 550 550 550 550 550 550 550 ...
#&gt;  $ title                             : chr  "Moving pictures of the human microbiome" "Moving pictures of the human microbiome" "Moving pictures of the human microbiome" "Moving pictures of the human microbiome" ...
#&gt;  $ principal_investigator            : chr  "Rob Knight" "Rob Knight" "Rob Knight" "Rob Knight" ...
#&gt;  $ doi                               : chr  "10.1186/gb-2011-12-5-r50" "10.1186/gb-2011-12-5-r50" "10.1186/gb-2011-12-5-r50" "10.1186/gb-2011-12-5-r50" ...
#&gt;  $ ebi_accession                     : chr  "ERP021896" "ERP021896" "ERP021896" "ERP021896" ...
#&gt;  $ target_gene                       : chr  "16S rRNA" "16S rRNA" "16S rRNA" "16S rRNA" ...
#&gt;  $ target_subfragment                : chr  "V4" "V4" "V4" "V4" ...
#&gt;  $ pcr_primers                       : chr  "FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT" "FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT" "FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT" "FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT" ...
#&gt;  $ illumina_technology               : chr  "HiSeq" "HiSeq" "HiSeq" "HiSeq" ...
#&gt;  $ extraction_center                 : chr  "CCME-Boulder" "CCME-Boulder" "CCME-Boulder" "CCME-Boulder" ...
#&gt;  $ run_center                        : chr  "CGS-GL" "CGS-GL" "CGS-GL" "CGS-GL" ...
#&gt;  $ run_date                          : chr  "8/27/10" "8/27/10" "8/27/10" "8/27/10" ...
#&gt;  $ read_length_bp                    : int  132 132 132 132 132 132 132 132 132 132 ...
#&gt;  $ sequences_split_libraries         : int  33383 40944 35636 46992 30131 29907 40565 49669 16954 20378 ...
#&gt;  $ observations_closed_ref_greengenes: int  32153 39472 34550 43925 29179 29034 39123 47800 15824 18484 ...
#&gt;  $ observations_closed_ref_silva     : int  32453 39929 34666 43852 29553 29310 39200 48154 16107 19185 ...
#&gt;  $ observations_open_ref_greengenes  : int  33337 40870 35599 46875 30094 29865 40433 49582 16870 20280 ...
#&gt;  $ observations_deblur_90bp          : int  22567 27871 24134 30041 21132 20875 25919 32962 10838 13559 ...
#&gt;  $ observations_deblur_100bp         : int  22160 27191 23686 29264 20643 20401 25653 32301 10983 13608 ...
#&gt;  $ observations_deblur_150bp         : int  1043 1272 1161 1974 603 572 4033 1602 391 913 ...
#&gt;  $ emp_release1                      : logi  TRUE TRUE TRUE TRUE TRUE TRUE ...
#&gt;  $ qc_filtered                       : logi  TRUE TRUE TRUE TRUE TRUE TRUE ...
#&gt;  $ subset_10k                        : logi  TRUE TRUE TRUE TRUE TRUE TRUE ...
#&gt;  $ subset_5k                         : logi  TRUE TRUE TRUE TRUE TRUE TRUE ...
#&gt;  $ subset_2k                         : logi  TRUE TRUE TRUE TRUE TRUE TRUE ...
#&gt;  $ sample_taxid                      : int  408170 408170 408170 408170 408170 408170 408170 408170 539655 539655 ...
#&gt;  $ sample_scientific_name            : chr  "human gut metagenome" "human gut metagenome" "human gut metagenome" "human gut metagenome" ...
#&gt;  $ host_taxid                        : int  9606 9606 9606 9606 9606 9606 9606 9606 9606 9606 ...
#&gt;  $ host_common_name_provided         : chr  "human" "human" "human" "human" ...
#&gt;  $ host_common_name                  : chr  "human" "human" "human" "human" ...
#&gt;  $ host_scientific_name              : chr  "Homo sapiens" "Homo sapiens" "Homo sapiens" "Homo sapiens" ...
#&gt;  $ host_superkingdom                 : chr  "sk__Eukaryota" "sk__Eukaryota" "sk__Eukaryota" "sk__Eukaryota" ...
#&gt;  $ host_kingdom                      : chr  "k__Metazoa" "k__Metazoa" "k__Metazoa" "k__Metazoa" ...
#&gt;  $ host_phylum                       : chr  "p__Chordata" "p__Chordata" "p__Chordata" "p__Chordata" ...
#&gt;  $ host_class                        : chr  "c__Mammalia" "c__Mammalia" "c__Mammalia" "c__Mammalia" ...
#&gt;  $ host_order                        : chr  "o__Primates" "o__Primates" "o__Primates" "o__Primates" ...
#&gt;  $ host_family                       : chr  "f__Hominidae" "f__Hominidae" "f__Hominidae" "f__Hominidae" ...
#&gt;  $ host_genus                        : chr  "g__Homo" "g__Homo" "g__Homo" "g__Homo" ...
#&gt;  $ host_species                      : chr  "s__Homo_sapiens" "s__Homo_sapiens" "s__Homo_sapiens" "s__Homo_sapiens" ...
#&gt;  $ collection_timestamp              : POSIXct, format: "2009-03-29" "2009-04-06" ...
#&gt;  $ country                           : chr  "GAZ:United States of America" "GAZ:United States of America" "GAZ:United States of America" "GAZ:United States of America" ...
#&gt;  $ latitude_deg                      : num  40 40 40 40 40 ...
#&gt;  $ longitude_deg                     : num  -105 -105 -105 -105 -105 ...
#&gt;  $ depth_m                           : num  0 0 0 0 0 0 0 0 0 0 ...
#&gt;  $ altitude_m                        : num  0 0 0 0 0 0 0 0 0 0 ...
#&gt;  $ elevation_m                       : num  1624 1624 1624 1624 1624 ...
#&gt;  $ env_biome                         : chr  "urban biome" "urban biome" "urban biome" "urban biome" ...
#&gt;  $ env_feature                       : chr  "human-associated habitat" "human-associated habitat" "human-associated habitat" "human-associated habitat" ...
#&gt;  $ env_material                      : chr  "feces" "feces" "feces" "feces" ...
#&gt;  $ envo_biome_0                      : chr  "biome" "biome" "biome" "biome" ...
#&gt;  $ envo_biome_1                      : chr  "terrestrial biome" "terrestrial biome" "terrestrial biome" "terrestrial biome" ...
#&gt;  $ envo_biome_2                      : chr  "anthropogenic terrestrial biome" "anthropogenic terrestrial biome" "anthropogenic terrestrial biome" "anthropogenic terrestrial biome" ...
#&gt;  $ envo_biome_3                      : chr  "dense settlement biome" "dense settlement biome" "dense settlement biome" "dense settlement biome" ...
#&gt;  $ envo_biome_4                      : chr  "urban biome" "urban biome" "urban biome" "urban biome" ...
#&gt;  $ envo_biome_5                      : chr  "" "" "" "" ...
#&gt;  $ empo_0                            : chr  "EMP sample" "EMP sample" "EMP sample" "EMP sample" ...
#&gt;  $ empo_1                            : chr  "Host-associated" "Host-associated" "Host-associated" "Host-associated" ...
#&gt;  $ empo_2                            : chr  "Animal" "Animal" "Animal" "Animal" ...
#&gt;  $ empo_3                            : chr  "Animal distal gut" "Animal distal gut" "Animal distal gut" "Animal distal gut" ...
#&gt;  $ adiv_observed_otus                : int  93 82 108 107 79 76 173 125 627 396 ...
#&gt;  $ adiv_chao1                        : num  114.4 116.5 135.1 126 94.5 ...
#&gt;  $ adiv_shannon                      : num  3.87 3.27 3.66 4.44 3.33 ...
#&gt;  $ adiv_faith_pd                     : num  12.5 10.7 14.2 12 11.8 ...
#&gt;  $ temperature_deg_c                 : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ ph                                : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ salinity_psu                      : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ oxygen_mg_per_l                   : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ phosphate_umol_per_l              : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ ammonium_umol_per_l               : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ nitrate_umol_per_l                : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  $ sulfate_umol_per_l                : num  NA NA NA NA NA NA NA NA NA NA ...
#&gt;  - attr(*, ".internal.selfref")=&lt;externalptr&gt; </div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by <a href='https://biodataanalysisgroup.github.io/team/'>Nikolaos Pechlivanis</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


